src/components/ComposerEditor/ComposerEditor.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 
-import CharacterLimitMainText from '~components/CharacterLimitMainText/CharacterLimitMainText'; // Importe o componente CharacterLimitMainText aqui
-import CustomTextArea from '~components/TextArea/TextArea'; // Importe o componente CustomTextArea aqui
+import CharacterLimitMainText from '../CharacterLimitMainText/CharacterLimitMainText'; // Importe o componente CharacterLimitMainText aqui
+import CustomTextArea from '../TextArea/TextArea'; // Importe o componente CustomTextArea aqui
```

Comment:
Vale sempre olhar o código que está sendo mexido. Nesse caso esses comentários nao eram necessários, pode remover.

```suggestion
import CharacterLimitMainText from '../CharacterLimitMainText/CharacterLimitMainText'; 
import CustomTextArea from '../TextArea/TextArea'; 
```